packages/form-helpers/README.md

```diff
@@ -34,9 +34,9 @@ const form = document.querySelector('#someForm');
 const input = document.querySelector('input');
 
 input.addEventListener( 'keypress', ($event) => {
-  if($event.keyCode === 13) { // Enter key
+  if($event.code === 'Enter') {
     submit(form); // submit event is emitted, and form's submit() method is called if the `submit` event is not `defaultPrevented`
-    console.log(submitted) // submitted will be false if the input doesn't have a value AND is required
+    console.log(submitted) // submitHandler will not be called if the input doesn't have a value AND is required
```

Comment:
This might not be accurate, but basically the previous copy is confusing. It says `submitted` will be false based on the input, but you only see `submitted` get updated via form submit callback?

Reply:
@calebdwilliams seems there's some nuance here that could be more clear. Both are "right", upon a close reading, but neither seem "easy". I lean a little closer to this suggestion, at which point we maybe don't need `submitted` at all, but want to make sure I'm not missing anything.

Would it make sense to pair this with something in `submitHandler` about it only getting called when the form is "valid"?

Reply:
@Westbrook, yes, that's the basic idea. The `submit` function will walk through the validation steps first before triggering the submit flow. However, it is probably best to note that this is really only necessary until Safari 16 which now ships with `HTMLFormElement.prototype.requestSubmit`. I'll merge this for now and look at rewriting this soon for more clarity.